src/App.js

```diff
@@ -1,28 +1,140 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+import React, { Component} from 'react';
 import './App.css';
+import {Layout, Header, Navigation, Drawer, Content, Footer,FooterSection, FooterDropDownSection, FooterLinkList} from 'react-mdl';
+import MainRoute from './routes/mainroute';
+import {Link} from 'react-router-dom';
+import {Badge, Icon} from 'react-mdl';
+import BuildFavBadge from './components/complibrary/buildfavbadge';
+import {Tooltip} from 'reactstrap';
 
+var fs = require('fs');
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
-  }
+
+    constructor(props){
+        super(props);
+        this.state = {
+            displayFavBadge: "",
+
+       }
+    }
+
+    //Build the fav badge and count alone. No setting state.
+    buildFavoriteBadge(){
```

Comment:
❗there should be a configuration with https://prettier.io or https://eslint.org/ or something similar to easily format the code for consistently